templates/compose/postgresus.yaml

```
@@ -0,0 +1,20 @@
+# documentation: https://postgresus.com
+# slogan: Postgresus is a free, open source and self-hosted tool to backup PostgreSQL.
+# category: devtools
+# tags: postgres,backup,self-hosted,open-source
+# logo: svgs/postgresus.svg
+# port: 4005
+
+services:
+  postgresus:
+    image: rostislavdugin/postgresus:latest
```

Please pin to a static version rather than 'latest' to prevent the template from breaking due to upstream changes.

```suggestion
    image: 'rostislavdugin/postgresus:7fb59bb5d02fbaf856b0bcfc7a0786575818b96f' # Released on Sep 30, 2025
```

---

Used static version.